Dockerfile.ubuntu

```diff
@@ -0,0 +1,91 @@
+# Stage 1: Builder stage
```

Comment:
Could you explain why we have two Dockerfiles here? When do we need the `.ubuntu` one? Might be good to add to the a README or add a comment somewhere in the file as it might be non-obvious.

Reply:
I added explanation about this in the README.